lib/active_merchant/billing/gateways/borgun.rb

```diff
@@ -130,7 +130,6 @@ def commit(action, post)
         post[:Processor] = @options[:processor]
         post[:MerchantID] = @options[:merchant_id]
 
-        url = (test? ? test_url : live_url)
```

Comment:
This one's fun. You'd think this should fail, given that the `url` _method_ is called two lines later, but somehow this works:

```ruby
irb(main):002:0> class Foo
irb(main):003:1>   def foo
irb(main):004:2>     u = 'hello'
irb(main):005:2>     u('goodbye')
irb(main):006:2>   end
irb(main):007:1>   def u(greeting)
irb(main):008:2>     puts greeting
irb(main):009:2>   end
irb(main):010:1> end
=> :u
irb(main):011:0> Foo.new.foo
goodbye
```

I'd have to look at the compiler to figure out what's going on here, because I have no idea.